fury/ui/containers.py

```diff
@@ -220,6 +255,73 @@ def re_align(self, window_size_change):
             msg = "You can only left-align or right-align objects in a panel."
             raise ValueError(msg)
 
+    def update_border_coords(self):
+        """Update the coordinates of the borders
+        """
+        self.border_coords = {'left': (0., 0.),
+                              'right': (1., 0.),
+                              'top': (0., 1.),
+                              'bottom': (0., 0.)}
+
+        for key in self.borders.keys():
+            self.update_element(self.borders[key], self.border_coords[key])
+
+    @property
+    def border_color(self):
+        labels = ['left', 'right', 'top', 'bottom']
+        return [self.borders[label].color for label in labels]
+
+    @border_color.setter
+    def border_color(self, label_color):
+        """Set the color of a specific border
```

Comment:
`side` instead of `label`. Reduces confusion.

Reply:
Also, Dragging on borders don't work.

Reply:
Both have been fixed.